File: carrera de caballos/web/js/model/Suit.js
/**
 * Palos de la baraja española (emojis).
 */
const Suit = {
  OROS: { key: 'OROS', displayName: 'Oros', symbol: '🪙' },
  COPAS: { key: 'COPAS', displayName: 'Copas', symbol: '🏆' },
  ESPADAS: { key: 'ESPADAS', displayName: 'Espadas', symbol: '⚔️' },
  BASTOS: { key: 'BASTOS', displayName: 'Bastos', symbol: '🪵' }
};

Suit.values = () => [Suit.OROS, Suit.COPAS, Suit.ESPADAS, Suit.BASTOS];

Suit.fromKey = (key) => Suit.values().find(s => s.key === key) || null;

window.Suit = Suit;

File: carrera de caballos/src/main/java/carrera/model/Suit.java
package carrera.model;

/**
 * Palos de la baraja española.
 * Usados para identificar caballos y movimiento en la pista.
 */
public enum Suit {
    OROS("Oros", "🪙"),
    COPAS("Copas", "🏆"),
    ESPADAS("Espadas", "⚔️"),
    BASTOS("Bastos", "🪵");

    private final String displayName;
    private final String symbol;

    Suit(String displayName, String symbol) {
        this.displayName = displayName;
        this.symbol = symbol;
    }

    public String getDisplayName() {
        return displayName;
    }

    public String getSymbol() {
        return symbol;
    }

    @Override
    public String toString() {
        return displayName;
    }
}
